packages/gatsby-theme-newrelic/src/components/Footer.js

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Icon from './Icon';
+import Logo from './Logo';
+import ExternalLink from './ExternalLink';
+import { graphql, useStaticQuery, Link } from 'gatsby';
+import { css } from '@emotion/core';
+
+const Footer = ({ fileRelativePath }) => {
+  const { site } = useStaticQuery(graphql`
+    query FooterQuery {
+      site {
+        siteMetadata {
+          repository
+        }
+        layout {
+          contentPadding
+        }
+      }
+    }
+  `);
+
+  const { siteMetadata, layout } = site;
+
+  return (
+    <footer
+      data-swiftype-index={false}
+      css={css`
+        align-items: center;
+        border-top: 1px solid var(--divider-color);
+        color: var(--secondary-text-color);
+        display: flex;
+        font-size: 0.75rem;
+        grid-area: footer;
+        justify-content: space-between;
+        margin-right: ${layout.contentPadding};
+        padding: ${layout.contentPadding} 0;
+        z-index: 1;
+
+        @media (max-width: 1080px) {
+          flex-direction: column;
+        }
+
+        a {
+          color: currentColor;
+        }
+      `}
+    >
+      <div
+        css={css`
+          display: flex;
+          align-items: center;
+
+          @media (max-width: 760px) {
+            display: block;
+          }
+        `}
+      >
+        <Link to="/">
+          <Logo />
+        </Link>
+        <div
+          css={css`
+            margin-left: 0.5rem;
+            padding-left: 0.5rem;
+            border-left: 1px dotted var(--border-color);
+
+            @media (max-width: 760px) {
+              border-left: none;
+              margin-left: 0;
+              padding: 0.5rem;
+            }
+          `}
+        >
+          Copyright &copy; 2020 New Relic Inc.
```

Comment:
According to the wireframe in the [MMF doc](https://docs.google.com/document/d/1jzFz-hls8Q-okkmm06x7GOPAWy1PxvJL72B-vWDOULs/edit#heading=h.i047btqfbnr9), it looks like this should be on its own row underneath the logo and the GitHub links, rather than right next to the logo. Could you update to match?